Bukkit/0096-Add-id-specific-map-API.patch

```diff
@@ -0,0 +1,29 @@
+From 20e09cc557749fbdd63037271703890d9145c58c Mon Sep 17 00:00:00 2001
+From: Pablete1234 <pabloherrerapalacio@gmail.com>
+Date: Mon, 9 Jan 2017 21:50:17 +0100
+Subject: [PATCH] Add id specific map API
+
+
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index c70b635..d349ae0 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -568,6 +568,15 @@ public interface Server extends PluginMessageRecipient, BukkitRuntime, tc.oc.min
+     public MapView createMap(World world);
+ 
+     /**
++     * Create a new map not following the internal per-world counter nms uses.
++     * Be caureful when using this method, as it may create map collisions, only use this for disposable worlds.
```

Comment:
I'm unsure on what will happen if 2 worlds have the same id, most likely the world will replace the old one with a the new one, or it may have other issues